src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/VerdictCategory.java

```diff
@@ -68,10 +89,108 @@ public String getVerdictDescription() {
      * @param obj the JSONObject.
      * @return a VerdictCategory.
      */
-    public static VerdictCategory createVerdictCategoryFromJSON(JSONObject obj) {
+    public static VerdictCategory createVerdictCategoryFromJSON(JSONObject obj, JSONObject topLevelJSON) {
```

Comment:
[minor] maybe rename it to `topLevelObj`? We already know from the type that it is a JSON.

Reply:
another [minor]: we know from the return value the it creates a `VerdictCategory`. What about renaming it to just `fromJSON`?

Reply:
Agreed, done.